sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Models/Sku.cs

```diff
@@ -32,16 +30,14 @@ public Sku()
         /// <summary>
         /// Initializes a new instance of the Sku class.
         /// </summary>
-        /// <param name="tier">The service tier</param>
-        /// <param name="name">The SKU name</param>
-        /// <param name="capacity">If the SKU supports scale out/in then the
-        /// capacity integer should be included. If scale out/in is not
-        /// possible for the resource this may be omitted.</param>
-        public Sku(string tier = default(string), string name = default(string), int? capacity = default(int?))
+        /// <param name="name">The name of the SKU, typically, a letter +
+        /// Number code, e.g. P3.</param>
+        /// <param name="tier">The tier or edition of the particular SKU, e.g.
+        /// Basic, Premium.</param>
+        public Sku(string name, string tier = default(string))
```

Comment:
this will result in a breaking change, please revert or have the breaking change team add the approval label to the spec pr

Reply:
reverted